docs/index.md

```diff
@@ -330,6 +339,8 @@ Currently, if multiple **kinesis_rejection** messages are applicable, multiple S
   - Stop free replan time dropoffs
 
 ## Pre-planning Data Validation
+is BD_no_area_price's message rendering correctly? i see "%" popping up on the site
```

Comment:
im looking at the site on safari if that helps/means/says anything

Reply:
% in the _Message_ is fine. % is to be rendered as values in the real use cases. We send it to show users this.